front/src/applications/stdcm/views/StdcmResultsV2.tsx

```diff
@@ -54,25 +39,26 @@ const StcdmResultsV2 = ({ mapCanvas, stdcmV2Results, pathProperties }: StcdmResu
     [pathProperties, stdcmV2Results]
   );
 
+  const manchetteSpaceTimeData: ProjectPathTrainResult[] = useMemo(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    () => {
+      const stdcmTimePosition = spaceTimeData.slice(-1);
```

Comment:
Why do you slice the last element ? It's the stdcm, we should display it too

Reply:
slice(-1) returns the last element

Reply:
My bad, I didn't tested when I wrote this comment. But I have now, and I can see that the other trains are not displayed, but I believe they should be

Reply:
Yes i agree but there is no space_time_curves for the other trains, they are empty in each simulations i did. The visualization is also still the same as previous result visualization.

Reply:
Hum maybe you have not tested right, because when I add a console.log in my scenario, I can see all that the other trains are projected, not empty and not displayed. I think you should remove the slice
![image](https://github.com/user-attachments/assets/562cc106-59bb-4df8-8fa1-410a5291c95e)